src/cli.c

```diff
@@ -425,74 +451,56 @@ static int import_keys(user_options_t *options)
         }
     }
 
-    if (!user) {
-        printf("Bridge username (email): ");
-        get_input(user_input);
-        num_chars = strlen(user_input);
-        user = calloc(num_chars + 1, sizeof(char));
-        if (!user) {
-            status = 1;
-            goto clear_variables;
-        }
-        memcpy(user, user_input, num_chars * sizeof(char));
-    }
+    if (!enc_access_str) {
+        enc_access_str = calloc(BUFSIZ, sizeof(char));
 
-    if (!pass) {
-        printf("Bridge password: ");
-        pass = calloc(BUFSIZ, sizeof(char));
-        if (!pass) {
-            status = 1;
-            goto clear_variables;
-        }
-        get_password(pass, '*');
+        printf("Enter your serialized encryption key (leave blank to generate new): ");
+        get_input(enc_access_str);
+        num_chars = strlen(enc_access_str);
         printf("\n");
-    }
 
-    if (!mnemonic) {
-        mnemonic_input = calloc(BUFSIZ, sizeof(char));
-        if (!mnemonic_input) {
-            status = 1;
-            goto clear_variables;
-        }
+        if (num_chars == 0) {
+            int last_result = -1;
+            for (int i = 0; i < 3; i++) {
+                last_result = get_password_verify("Enter a passphrase to generate "
+                                                  "a new root key for your "
+                                                  "encryption key: ", new_enc_passphrase, 0);
```

Comment:
IMHO, a better message would be: "Enter a passphrase to create a new serialized encryption access: "